src/components/four-up.test.tsx

```diff
@@ -25,6 +25,11 @@ jest.mock("../hooks/use-stores", () => ({
   }),
   useUIStore: () => ({
     showAdornments: true
+    // TODO: FIXME - doesn't seem like showAdornments is being used (since missing from UIModel)
+    // but useUIStore needs to be defined
```

Comment:
Yeah, you should try removing this and seeing if the test still passes.

Reply:
It seems like in this case no properties are required, and that things run with 

```ts
  useUIStore: () => ({}),
  usePersistentUIStore: () => ({})
```

but will not run without at least the above